Kernel/Arch/aarch64/PSCI.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2025, the SerenityOS developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+namespace Kernel::PSCI {
+
+bool psci_supported();
```

Comment:
We usually add the `is_` prefix to functions returning a `bool`. And, as it's already in the `PSCI` namespace, you could simply name that function `is_supported()`.